docs/metrics.md

```diff
@@ -0,0 +1,113 @@
+# METRICS
+
+## Data Analysis
+
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do people use this?
+
+What is the overall engagement of Snooze Tabs? **This is the standard Daily
+Active User (DAU) and Monthly Active User (MAU) analysis.** This captures data
+from the people who have the add-on installed, regardless of whether they are
+actively interacting with it.
+
+### Immediate Questions
+
+TBD
+
+### Follow-up Questions
+
+TBD
+
+## Data Collection
+
+### Server Side
+There is currently no server side component to Snooze Tabs.
+
+### Client Side
+Snooze Tabs will use Test Pilot's Telemetry wrapper with no batching of data.
```

Comment:
We'll modify this to say something about @_6a68 's broker once we have more details on it